lib/stylelint/plugin.rb

```diff
@@ -1,34 +1,117 @@
 # frozen_string_literal: true
 
+require "mkmf"
+require "json"
+
 module Danger
-  # This is your plugin class. Any attributes or methods you expose here will
-  # be available from within your Dangerfile.
-  #
-  # To be published on the Danger plugins site, you will need to have
-  # the public interface documented. Danger uses [YARD](http://yardoc.org/)
-  # for generating documentation from your plugin source, and you can verify
-  # by running `danger plugins lint` or `bundle exec rake spec`.
+  # Lint javascript files using [stylelint](http://stylelint.io/).
```

Comment:
Maybe

```suggestion
  # Lint stylesheet files using [stylelint](http://stylelint.io/).
```

Reply:
done in 4f636e6f6e7e07faf138ee91211bdca380637e61!